source/dbo/Tables/Power BI Reporting/Record_CaseData.sql

```diff
@@ -52,12 +52,15 @@
 	[MdrTreatmentDate] date NULL,
 	[EnhancedCaseManagement] [varchar] (10) NULL,
 	[EnhancedCaseManagementLevel] [nvarchar] (10) NULL,
+	[FirstPresentationSetting] [nvarchar] (25) NULL,
```

Comment:
I don't think this will be long enough - the free text healthcare setting description field is 100 characters in the database.